examples/concurrency/main.rs

```diff
@@ -4,28 +4,31 @@ use surrealdb_rs::StaticClient;
 use surrealdb_rs::Surreal;
 use tokio::sync::mpsc;
 
-static CLIENT: Surreal<WsClient> = Surreal::new();
+static DB: Surreal<WsClient> = Surreal::new();
 
 const NUM: usize = 100_000;
 
 #[tokio::main]
 async fn main() -> surrealdb_rs::Result<()> {
     tracing_subscriber::fmt::init();
 
-    CLIENT
-        .connect::<Ws>("localhost:8000")
+    DB.connect::<Ws>("localhost:8000")
```

Comment:
```rs
#![cfg(any(feature = "ws", feature = "mem"))]
#[cfg(feature = "ws")]
use surrealdb_rs::net::WsClient;
#[cfg(feature = "ws")]
use surrealdb_rs::protocol::Ws;
use surrealdb_rs::storage::Mem;
use surrealdb_rs::StaticClient;
use surrealdb_rs::Surreal;
use tokio::sync::mpsc;
#[cfg(feature = "ws")]
static DB: Surreal<WsClient> = Surreal::new();
#[cfg(any(
    feature = "mem",
    feature = "tikv",
    feature = "rocksdb",
    feature = "fdb"
))]
use surrealdb_rs::embedded::Db;
#[cfg(any(
    feature = "mem",
    feature = "tikv",
    feature = "rocksdb",
    feature = "fdb"
))]
static DB: Surreal<Db> = Surreal::new();

const NUM: usize = 100_000;

#[tokio::main]
async fn main() -> surrealdb_rs::Result<()> {
    tracing_subscriber::fmt::init();
    #[cfg(feature = "ws")]
    DB.connect::<Ws>("localhost:8000")
        .with_capacity(NUM)
        .await?;
    #[cfg(feature = "mem")]
    DB.connect::<Mem>(()).with_capacity(NUM).await?;
```

Probably the better one with cfg feature.

Reply:
Thanks for the suggestion. A separate example would be more readable. We can add more examples in a separate PR.